help/license.md

```diff
@@ -1,103 +1,73 @@
+
+<br/>![CC Zero](images/cc-zero.png){.mkd-img-left} [CC Zero](https://creativecommons.org/share-your-work/public-domain/cc0/) is a public dedication tool, which allows creators to give up their copyright and put their works into the worldwide public domain. CC0 allows reusers to distribute, remix, adapt, and build upon the material in any medium or format, with no conditions.
+
+_If you choose this license, you will no longer control the article’s copyright. A work distributed with CC Zero would not allow for copyright transfer, and this conflicts with many publishers’ requirements._
+
 
 Metadata license
------------------------------------------
-
-To the extent that the depositor or arxiv has a copyright interest in metadata accompanying the submission, a [Creative Commons CC0 1.0 Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/) will apply.
-
-Authority to submit
------------------------------------------
-
-arXiv cannot accept papers that contain material for which the depositor
-does not have the authority to submit or to agree to the license terms.
-This would include comments by referees (which may have separate
-copyright protection) and, of course, plagiarized material. Note that
-any single co-author normally has all needed authority to submit a paper
-to arXiv. Once publicly available, articles cannot be entirely removed,
-either at the request of the submitting author or of any co-author.
-
-
-Licenses granted are irrevocable
---------------------------------
-
-Authors should take care to upload an article *only if they are certain*
-that they will not later wish to publish it in a journal that prohibits
-prior distribution on an e-print server. *arXiv will not remove an
-announced article to comply with such a journal policy -- the license
-granted on submission is irrevocable.* However, granting rights for
-arXiv to distribute an article does not preclude later copyright
-assignment. Authors are thus free to publish submissions that already
-appear on arXiv. Authors may wish to inform the journal publisher that a
-prior non-exclusive license exists before transferring copyright or
-granting a publication license. Please check the policies of any
-potential publication venue before uploading to arXiv. (For the policy
-information of many publishers, see the
-[SHERPA/RoMEO](http://www.sherpa.ac.uk/romeo.php) site.)
+----------------
+A [Creative Commons CC0 1.0 Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/) will apply to all metadata.
 
 Copyright notices
 -----------------
+Often, PDFs found on publisher websites or supplied as proofs are the property of the publisher, which often owns the copyright and/or licenses their use. Submission of PDFs with copyright statements which prohibit or impair arXiv's redistribution license will be rejected. If explicit permission is given for open access redistribution, these are permissible.

```

Comment:
```suggestion
Often, PDFs found on publisher websites or supplied as proofs are the property of the publisher, which often owns the copyright and/or licenses their use. Submission of PDFs with copyright statements which prohibit or impair arXiv's redistribution license will be rejected. If explicit permission is given for open access redistribution, these are permissible.
Users should contact arXiv administrators at help@arxiv.org if they have questions.
```
This was added recently as we had out of band communication. Would prefer to keep it for now.